### PR TITLE
01-numpy.md: change weight to avoid numerical errors

### PR DESCRIPTION
**Pull request by joshkyh**
Proposed fix to floating point print by through native function format #328 

---
_edited by @maxim-belkin to remove PR template_

### CODE REVIEW

**Comment by MikeTrizna** [comment]
I agree with @maxim-belkin that it would be simpler to change `weight` from 55 to 60 (or any other integer that doesn't produce this behavior). 

I've personally run into this behavior early in a Python workshop, and having to explain what was happening caused a major delay AND left a few participants concerned that they would be better off continuing to do calculations in Excel...

**Comment by maxim-belkin** [comment]
Looks like @tbekolay suggested the same solution a while back here: https://github.com/swcarpentry/python-novice-inflammation/pull/339#issuecomment-339103601

@joshkyh can you update your pull request to change the weight from 55 to 60?

**Comment by maxim-belkin** [comment]
Also, is this part of instructor checkout?

**Comment by joshkyh** [comment]
@maxim-belkin done - pull request approach changed to simply changing the weight to 60 :) Not sure about your question on "is this part of instructor checkout". Is this question meant for me?

**Comment by joshkyh** [comment]
Hi @maxim-belkin Apologies, I should have looked through the rest of the file in detail. I have changed the rest of the code that referenced the original weight and changed it accordingly.

I did some search with reference to "instructor checkout" procedure, I am not part of any instructor training programme and does not require this commit to be logged against my progress - just a member of the public. :)

Thanks,
Joshua

File: _episodes/01-numpy.md [hunk 1]
```diff
@@ -53,19 +53,21 @@ print(weight_kg)
...
-print('weight in pounds:', 2.2 * weight_kg)
+weight_lb = 2.2 * weight_kg
+
+print('weight in pounds:', weight_lb)
 ~~~
 {: .python}
 
 ~~~
-weight in pounds: 121.0
+weight in pounds: 120.0
```
Comment (tbekolay) [inline _episodes/01-numpy.md]: I believe this should be `132.0.`

**Comment by joshkyh** [comment]
@tbekolay yes you're right, my bad. Thanks for pointing out.

**Pull request closed without merging**